src/react/reconcilation.js

```diff
@@ -482,6 +485,9 @@ export class Reconciler {
   }
 
   _hasReferenceForContextNode(contextNode: ObjectValue | AbstractObjectValue): boolean {
```

Comment:
It's confusing to me that a function says it "has a reference" just by looking at flag. Technically it doesn't even look at the "references" field.

Does the function need to be renamed to better reflect its new meaning?

Reply:
It does look up reference counts, unless it's the root, in which case it doesn't need to as there can be no parent components with context in.

Reply:
Happy to rename if you have  a better name though.

Reply:
What does it help your code *determine* (as opposed to what it does)? That should be the name. :-)

Reply:
For example `_isContextValueKnown()` or something.

Reply:
`_isContextValueKnown` sounds good, if you are fixing this PR, would you mind making this change too please? I'm sorting out the conditional PR :)